test/fgpu/test_engine.py

```diff
@@ -956,6 +956,40 @@ async def test_dig_clip_cnt_sensors(
         assert [r.value for r in sensor_update_dict[sensors[1].name]] == [0, 0] + [10000] * 7
         assert [r.timestamp for r in sensor_update_dict[sensors[1].name]] == expected_timestamps
 
+    # It's easier to use a constant voltage. Also need to check the case were
+    # the input power is zero.
+    @pytest.mark.parametrize("inout", [(0, np.finfo(np.float64).min), (64, -15.034518566376697)])
```

Comment:
How about
```suggestion
    @pytest.mark.parametrize("input_voltage,output_power_dbfs", [(0, np.finfo(np.float64).min), (64, -15.034518566376697)])
```
so that pytest will destructure the tuple for you.

Reply:
That looks neater, I like it.